pkg/controller/helpers/helpers.go

```diff
@@ -82,7 +82,8 @@ func ReconcileDeployment(owner metav1.Object, deployment *appsv1.Deployment, req
 			for k, v := range deployment.Labels {
 				toUpdate.Labels[k] = v
 			}
-			return nil
+			// Set the owner and controller
```

Comment:
Github is not letting me add comments up there, but doesn't L68-L71 become unnecessary after this change?

Reply:
I thought so too, but I’ve seen tests fail with those lines removed; I wanted to get the fix in, I’ll look into removing the theoretically redundant lines in a follow-up.